### PR TITLE
HS-945: Invoke Scan task after Discovery

### PR DESCRIPTION
**Pull request by cgorantla**
## Description
Invoke Scan task after Discovery.

## Jira link(s)
- https://issues.opennms.org/browse/HS-945

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

**Review by BenjaminJ — COMMENTED** [review]
LGTM

**Review by yangli2014 — APPROVED** [review]
Tested and works as expected.

File: inventory/src/main/java/org/opennms/horizon/inventory/service/NodeService.java [hunk 1]
```diff
@@ -234,4 +234,22 @@ private void removeAssociatedTags(Node node) {
             tag.getNodes().remove(node);
         }
     }
+
+    public void sendNewNodeTaskSetAsync(Node node, String tenantId) {
+        executorService.execute(() -> sendTaskSetsToMinion(node, tenantId));
+    }
+
+    private void sendTaskSetsToMinion(Node node, String tenantId) {
```
Comment (yangli2014) [inline inventory/src/main/java/org/opennms/horizon/inventory/service/NodeService.java:242]: This method can be removed.

Comment (cgorantla) [reply inventory/src/main/java/org/opennms/horizon/inventory/service/NodeService.java:242]: Remove unused method in NodeGrpcService

**Comment by cgorantla** [comment]
Will wait for pipleline to run End-to-end tests as it was failing earlier

**Review by BenjaminJ — APPROVED** [review]

**Pull request merged**